docs/repo-docs/guides/migrating-from-nx.mdx

```diff
@@ -433,7 +366,7 @@ We encourage incremental migration, meaning you will have both of Nx and Turbore
 
 ### Installing dependencies where they're used
 
-Turborepo recommends [installing packages where they're used](/repo/docs/crafting-your-repository/managing-dependencies#best-practices-for-dependency-installation) to improve cache hit ratios, help dependency pruning capability, and clarify for developers which dependencies are meant for which packages. This is different from the Nx strategy, where all dependencies are installed at the root of the repository, making all dependencies available to all packages in the workspace.
+Turborepo recommends [installing packages where they're used](/repo/docs/crafting-your-repository/managing-dependencies#best-practices-for-dependency-installation) to improve cache hit ratios, help dependency pruning capability, and clarify for developers which dependencies are meant for which packages.
```

Comment:
```suggestion
Turborepo recommends [installing packages where they're used](/repo/docs/crafting-your-repository/managing-dependencies#best-practices-for-dependency-installation) to improve cache hit ratios, help dependency pruning capability, and clarify for developers which dependencies are meant for which packages. This is different from the recommended Nx strategy, where all dependencies are installed at the root of the repository, making all dependencies available to all packages in the workspace.
```

Reply:
Just added "recommended" here. Thinking that helps to clarify? More context in [this thread](https://github.com/vercel/turborepo/pull/10003#discussion_r1962310092) as well.

Reply:
Thanks, I responded on the the thread